src/main/resources/templates/macros/report/tags.vm

```diff
@@ -2,7 +2,7 @@
 
 <div class="tags indention">
   #foreach($tag in $tags)
-     <a href="$tag.getFileName()">$tag.getName()</a>
+     <a class="btn btn-default btn-xs" href="$tag.getFileName()">$tag.getName()</a>
```

Comment:
On the other hand I'm not sure about using button for something that is pure link which drives user to another page - this is what link us used for and I believe user expectation are clear when having link instead of button - like at the summary page